src/main/java/kappzzang/jeongsan/service/PersonalExpenseService.java

```diff
@@ -156,9 +156,14 @@ private void checkRequestQuantityValidity(ItemInfo itemInfo, Item item) {
     private CalculatedPrice calculatedPrice(List<PersonalExpense> personalExpenses, Item item,
         int requestQuantity) {
 
+        int newPersonalUnitPrice;
         int totalQuantity = personalExpenses.stream().mapToInt(PersonalExpense::getQuantity).sum()
             + requestQuantity;
-        int newPersonalUnitPrice = item.getTotalPrice() / totalQuantity;
+        if (totalQuantity > item.getQuantity()) {
+            newPersonalUnitPrice = item.getTotalPrice() / totalQuantity;
+        } else {
+            newPersonalUnitPrice = item.getUnitPrice();
+        }
```

Comment:
P3: else를 지양하기 위해선 
```suggestion 
        int newPersonalUnitPrice = item.getUnitPrice();
        int totalQuantity = personalExpenses.stream().mapToInt(PersonalExpense::getQuantity).sum()
            + requestQuantity;
        if (totalQuantity > item.getQuantity()) {
            newPersonalUnitPrice = item.getTotalPrice() / totalQuantity;
        }
```
도 좋겠네요~

Reply:
반영 했습니다~